lint-configs/pyproject.toml

```
@@ -1,8 +1,91 @@
-[tool.black]
-line-length = 145
-target-version = ['py39']
-verbose = true
-
-[tool.isort]
-line_length = 145
-profile = "black"
+# Ruff linting
+[tool.ruff]
+line-length = 100 # Changed from 145 (more standard)
+target-version = "py39"
+
+[tool.ruff.lint]
+select = [
+    "E",
+    "W",
+    "F",
+    "I",
+    "C90",
+    "A",
+    "UP",
+    "B",
+    "C4",
+    "RET",
+    "SIM",
+    "RUF",
+    "PLC",
+    "PLE",
+    "PLW",
+    "PT",
+    "T201",
+]
+
+# Ignore same relevant rules as we did to with tox.ini
+ignore = [
+    "F403",
+    "F405",
+    "E731",
+    "E201",
+    "E202",
+    "E203",
+    "E701",
+    "E722",
+    "D",
+]
+
+# Keep complexity below 28
+[tool.ruff.lint.mccabe]
+max-complexity = 28
+
+[tool.ruff.lint.isort]
+combine-as-imports = true
+lines-after-imports = 2
+
+[tool.ruff.format]
+quote-style = "double"
+indent-style = "space"
+skip-magic-trailing-comma = false
+line-ending = "auto"
+
+# HTML linting
+[tool.djlint]
+profile = "handlebars"
```

Shouldn't this be django?
```suggestion
profile = "django"
```

---

Yeah this should be, not sure we actually need this but for consistency we'll include